.github/workflows/gh-ci.yaml

```diff
@@ -37,6 +37,11 @@ jobs:
               python-version: "3.11"
               full-deps: true
               codecov: true
+            - name: macOS_M1
+              os: macOS-14
+              python-version: "3.11"
```

Comment:
why not 3.12?

Reply:
Used the same version above. Is 3.12 preferred?

Reply:
Yeah - maybe bump both up? I think we held off on updating because there were missing py3,12 "full deps" in macos space, but that _should_ be fine now?